src/parser/flow_lexer.ml

```diff
@@ -58,6 +58,14 @@ let scinumber = [%sedlex.regexp?
 let wholenumber = [%sedlex.regexp? underscored_digit, Opt '.']
 let floatnumber = [%sedlex.regexp? Opt underscored_digit, '.', underscored_decimal]
 
+let binbigint = [%sedlex.regexp? binnumber, 'n']
+let octbigint = [%sedlex.regexp? octnumber, 'n']
+let legacyoctbigint = [%sedlex.regexp? legacyoctnumber, 'n']
+let hexbigint = [%sedlex.regexp? hexnumber, 'n']
+let scibigint = [%sedlex.regexp? decintlit, ('e' | 'E'), Opt ('-' | '+'), underscored_digit, 'n']
+let wholebigint = [%sedlex.regexp? underscored_digit, 'n']
+let floatbigint = [%sedlex.regexp? Opt underscored_digit, '.', underscored_decimal, 'n']
```

Comment:
Can this be `[%sedlex.regexp? floatnumber, 'n']`?

Or maybe `[%sedlex.regexp? (floatnumber | ('.', underscored_decimal)), 'n']` to outlaw `.1n`?

Reply:
Doesn't Opt already outlaw this?

Reply:
Sorry, typo. I meant outlaw `1.n`